web/packages/teleport/src/Roles/Roles.tsx

```diff
@@ -93,9 +93,10 @@ export function Roles(props: State) {
               Role-based access control
             </Text>
             <Text typography="subtitle1" mb={3}>
-              Kubernetes and SSH authentication in one place. A Teleport role
-              can be assigned based on user identity when used with single
-              sign-on (SSO).
+              Teleport Role-based access control (RBAC) provides fine-grained
+              control over who can access resources and in which contexts.
+              A Teleport role can be assigned based on user identity when used
+              with single sign-on (SSO).
```

Comment:
@alexfornuto @stevenGravy could we insert "automatically"? My brain struggled with that second sentence far more than I care to admit: 🤔 ... "Isn't it based on identity if I manually assign a role to a user?"

`A Teleport role can be assigned automatically based on user identity when used with single sign-on (SSO).`

Even better, we could link "single sign-on (SSO)" to the Auth Connectors page so that users know where they can add an identity provider... especially since we call it "single sign-on" here but "Auth Connectors" in the nav.